src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -5569,4 +5534,73 @@ export class StudyViewPageStore {
         onError: error => Promise.resolve([]),
         default: [],
     });
+
+    // return prefix
+    readonly survivalClinicalAttributesPrefix = remoteData({
+        await: () => [this.clinicalAttributes],
+        invoke: () => {
+            const attributes = getSurvivalAttributes(
+                this.clinicalAttributes.result!
+            );
+            let pairedAttributes: {
+                [prefix: string]: { status: string; months: string };
+            } = {};
+            let attributesPrefix: string[] = [];
+            // get paired attributes
+            attributes.map(attribute => {
+                let prefix = attribute.substring(
```

Comment:
you should use regexp and string.match for this.  you get case insensitivity and more flexibility

Reply:
@alisman Basically this part is to get the attributes' prefix. For example get `OS` from `OS_STATUS`, not sure how can I do this with string.match. I think we can do string.match for `attribute.indexOf('_STATUS')` below, but it doesn't make much sense since all attributes name are in uppercase